srcpkgs/xwinwrap/template

```diff
@@ -3,13 +3,13 @@ pkgname=xwinwrap
 version=0.9
 revision=1
 build_style=gnu-makefile
-makedepends="libXrender-devel"
+makedepends="libXrender-devel grep"
```

Comment:
why is grep needed?

Reply:
fails with grep not found.  ive re-bootstrapped masterdir. always get grep not found errors.
xwinwrap-0.9_1: running pre-pkg hook: 04-generate-runtime-deps ...
/void-packages/common/hooks/pre-pkg/04-generate-runtime-deps.sh: line 96: grep: command not found
/void-packages/common/hooks/pre-pkg/04-generate-runtime-deps.sh: line 97: grep: command not found
   SONAME: libX11.so.6 <-> UNKNOWN PKG PLEASE FIX!
/void-packages/common/hooks/pre-pkg/04-generate-runtime-deps.sh: line 96: grep: command not found
/void-packages/common/hooks/pre-pkg/04-generate-runtime-deps.sh: line 97: grep: command not found
   SONAME: libXext.so.6 <-> UNKNOWN PKG PLEASE FIX!
/void-packages/common/hooks/pre-pkg/04-generate-runtime-deps.sh: line 96: grep: command not found
/void-packages/common/hooks/pre-pkg/04-generate-runtime-deps.sh: line 97: grep: command not found

Reply:
try removing grep from build dependencies, zap, binary-bootstrap and build again

Reply:
zap , re-bootstrapped worked..
grep removed from commit.